docs/faq/troubleshooting/not-a-member-of-the-organization.md

```diff
@@ -2,11 +2,33 @@
 
 When you see this message, it means that Codacy can't analyze a commit because the associated email address doesn't belong to any Codacy member or author.
 
-There may be different reasons for this to happen:
+You can check which email address is associated with a commit by hovering the cursor on the name of the contributor on the page for the commit:
 
--   The user making the commit hasn't [signed in to Codacy and joined the organization](../../getting-started/getting-started-with-codacy.md) yet. Or, if the user doesn't belong to your organization, [add the user as an author](../../organizations/adding-and-managing-authors.md) instead.
+![Checking the email address for a commit](images/not-a-member-of-the-organization-commit.png)
 
--   The commit email address isn't associated with the account of a Codacy user. Codacy automatically associates the email addresses from the Git provider accounts to the Codacy accounts when users sign in to Codacy. Make sure that the user configures the missing email address on their Git provider account, and that the user logs in again on Codacy for the change to take effect.
+To verify which email addresses are associated with the Codacy account, the user must click on their avatar on the top right-hand corner, select **Your account**, and open the page **Emails**:
 
-!!! note
-    Please note that Codacy doesn't allow adding email addresses ending with `.local`. In this case, please use the email address that's associated with your Codacy account instead.
+![Email addresses associated with a user account](images/not-a-member-of-the-organization-account.png)
```

Comment:
I decided to pull these instructions on how to check the email address of a commit and which email addresses belong to a user account to the top since this can be used to troubleshoot all scenarios described below.